README(site plan).md

```diff
@@ -0,0 +1,10 @@
+Overview (b8-8.5 audis)
+    The practical useage of this website would be for one place for all of audi parts aftermaket and OEM for the cars. it would be a static site that will show the lowest price that a peticular brand is offering 
+Goals
+    The main goal of this website will have all after market and OEM parts for cars on one page
```

Comment:
How do you plan to list "all" aftermarket and OEM parts? You should have a couple more goals. Please reference my sample planning document for a couple of examples.